src/bp/nlu-core/training-worker-queue/index.ts

```diff
@@ -262,6 +262,7 @@ if (cluster.isWorker && process.env.WORKER_TYPE === WORKER_TYPES.TRAINING) {
         tools.seededLodashProvider.resetSeed()
       }
 
+      // TODO: send multiple packet when output is to big
```

Comment:
does that happen ?

Reply:
Well it will break if model is about a 1Gb, but I've never experienced such big models.

In this PR, clinc150 (with 20 utt per intents) produces a model of about 0.2 Gb.

I feel like we're good for a while with this problem, but we'll need to fix it eventually.